x/accesscontrol/handler.go

```diff
@@ -0,0 +1,28 @@
+package accesscontrol
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/cosmos/cosmos-sdk/x/accesscontrol/keeper"
+	"github.com/cosmos/cosmos-sdk/x/accesscontrol/types"
+)
+
+func HandleMsgUpdateResourceDependencyMappingProposal(ctx sdk.Context, k *keeper.Keeper, p *types.MsgUpdateResourceDependencyMappingProposal) error {
+	for _, resourceDepMapping := range p.MessageDependencyMapping {
+		k.SetResourceDependencyMapping(ctx, resourceDepMapping)
```

Comment:
do we do any checks over `resourceDepMapping`?

Reply:
Mainly just the proto serialization checks -- all the fields that are not free form are ENUMs so the proto serialization should also cover those cases